### PR TITLE
Feature/delete confirm

### PR DESCRIPTION
**Pull request by Omriruvio**
Add a feature that pops a confirmation box upon attempting to delete card

### CODE REVIEW

File: blocks/confirm-banner/__warpper/_active/confirm-banner__wrapper_active.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.confirm-banner__wrapper_active {
+  visibility: visible;
+  clip-path: polygon(-11.05% -15.69%, 109.47% -14.13%, 109.21% 117.25%, -10.79% 106.31%);
```
Comment (Sashikbear) [inline blocks/confirm-banner/__warpper/_active/confirm-banner__wrapper_active.css:3]: Really nice idea with the clip-path

File: blocks/confirm-banner/__warpper/confirm-banner__wrapper.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.confirm-banner__wrapper {
+  width: 90%;
+  text-align: center;
+  background-color: #ae3838;
+  padding: 10px;
+  border-radius: 15px;
+  visibility: hidden;
+  clip-path: polygon(50% 50%, 50% 50%, 50% 50%, 50% 50%);
+  transition: clip-path 0.3s, visibility 0.3s;
```
Comment (Sashikbear) [inline blocks/confirm-banner/__warpper/confirm-banner__wrapper.css:12]: Cool transition, I like how the visibility affects just the target card, not the whole window

**Review by Sashikbear — APPROVED** [review]

File: blocks/confirm-banner/__warpper/confirm-banner__wrapper.css (hunk 2 above)
Comment (Omriruvio) [reply blocks/confirm-banner/__warpper/confirm-banner__wrapper.css:12]: 😊

**Comment by Omriruvio** [comment]
fixes required before merge:
fix console error when selecting multiple cards to delete
fix behavior of creating new card displaying the delete message on creation when another delete query is active

**Pull request merged**